third_party/move/move-compiler-v2/tests/acquires-checker/access_ok.exp

```diff
@@ -0,0 +1,91 @@
+
+Diagnostics:
+error: access specifier not enabled. Only plain `acquires R` is enabled.
```

Comment:
This is a little confusing, since you're using `R` as a generic resource name, but that's not clear.  Perhaps adding a parenthetical "(for a resource named R)".  Or maybe
```
Only `acquires R` for a simple resource (struct) name `R` is enabled.
```

Reply:
Or maybe
```
Acquired resource must refer to a single `struct` name (e.g., `R` or `addr::Mod::R`).
```

Reply:
done